plural/terraform/aws/variables.tf

```diff
@@ -103,6 +103,26 @@ variable "single_az_node_groups" {
         }
       ]
     }
+    plural_worker_small = {
+      name = "plural-worker-small"
+      capacity_type = "ON_DEMAND"
+      min_capacity = 0
```

Comment:
we should have at least 1 as min capacity (there will always be some workers)

Reply:
(we should do this for the other group as well)

Reply:
done